src/command/mcl.zig

```diff
@@ -1896,6 +1959,186 @@ fn mclWaitRecoverSlider(params: [][]const u8) !void {
     }
 }
 
+/// Create a logging file for logging register values specified by
+/// the desired register and desired axis. Every successfull setup
+/// can be accessed through log_line variable.
+fn setLogRegisters(params: [][]const u8) !void {
+    const line_name: []const u8 = params[0];
+    const line_idx: usize = try matchLine(line_names, line_name);
+    const line: mcl.Line = mcl.lines[line_idx];
+
+    var log: RegisterLogging = undefined;
+    log.line_index = @intCast(line_idx);
+
+    // Iterate over axis list from parameters
+    var station_indices: []u8 = try allocator.alloc(u8, line.stations.len);
+    var idx: u8 = 0;
+    var axis_input_iterator = std.mem.tokenizeSequence(u8, params[1], ",");
+    outer: while (axis_input_iterator.next()) |token| {
+        const axis_index = try std.fmt.parseInt(mcl.Station.Id, token, 0) - 1;
+        if (axis_index < 0 or axis_index > line.axes.len - 1) {
+            return error.InvalidAxis;
+        }
+        const station_index: Station.Index = @intCast(axis_index / 3);
+        //TODO: Better checking than O(n)
+        for (station_indices) |item| {
+            if (station_index == item) continue :outer;
+        }
+        station_indices[idx] = station_index;
+        idx += 1;
+    }
+    log.station_indices = station_indices[0..idx];
+
+    // Iterate over register list from parameters
+    var reg_input_iterator = std.mem.tokenizeSequence(u8, params[2], ",");
+    while (reg_input_iterator.next()) |token| {
+        if (std.meta.stringToEnum(RegisterLogging.RegisterType, token)) |item| {
```

Comment:
You should support any casing for these register parameters. I recommend using `ascii`'s ignore case comparison with an `inline for` loop to turn the input into an `enum`.